robot_skills/src/robot_skills/force_sensor.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
```

Comment:
Only import the required module of numpy.

Reply:
Import as np is the convention used all over the internet.

Reply:
Yep the AS is correct. But why import whole numpy if you just need one function/class?